packages/react-native-bpk-component-text-input/src/styles.js

```diff
@@ -17,7 +17,8 @@
  */
 /* @flow */
 
-import { StyleSheet, I18nManager, type AnimatedValue } from 'react-native';
+import { StyleSheet, I18nManager } from 'react-native';
+import AnimatedValue from 'react-native/Libraries/Animated/src/nodes/AnimatedValue';
```

Comment:
Do you think there's any merit to us abstracting this and some other common types out somewhere, so we could then do something like:

```
import { AnimatedValue, otherStuff } from 'some-convenient-package';
```

I'm concerned that these paths are hard to find, e.g. `'react-native/Libraries/Animated/src/nodes/AnimatedValue'` and there's nothing to stop them changing in RN. If we abstract them and it does change, we can update it in one place.

Reply:
We can create our own libdef for it - but I reckon that's just putting a layer of indirection that would potentially mask errors if underlying types in react native change.

As for minimising the number of files we'd need to change if the file changes path or what not ... i'm a little indifferent tbh. Nothing a search + replace cant fix?

Reply:
Yeah fair enough - perfect being the enemy of good and all, let's just leave it as is.